
# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50  

# Position Controllers ---------------------------------------
j1_position_controller:
  type: effort_controllers/JointPositionController
  joint: j1
  pid: {p: 2, i: 0.6, d: 0.6}
j2_position_controller:
  type: effort_controllers/JointPositionController
  joint: j2
  pid: {p: 2, i: 0.6, d: 0.6}
j3_position_controller:
  type: effort_controllers/JointPositionController
  joint: j3
  pid: {p: 2, i: 0.6, d: 0.6}
j4_position_controller:
  type: effort_controllers/JointPositionController
  joint: j4
  pid: {p: 2, i: 0.6, d: 0.6}
j5_position_controller:
  type: effort_controllers/JointPositionController
  joint: j5
  pid: {p: 2, i: 0.6, d: 0.6}
j6_position_controller:
  type: effort_controllers/JointPositionController
  joint: j6
  pid: {p: 2, i: 0.6, d: 0.6}